gallery/plot_optical_flow.py

```diff
@@ -0,0 +1,105 @@
+"""
+==============================
+Optical Flow using Torchvision
+==============================
+
+Optical flow is a task consisting of estimating per pixel motion between two consecutive frames of a video.
+We aim to find the displacement of all image pixels and calculate their motion vectors.
+The following example illustrates how torchvision can be used in predicting as well as
+visualizing optical flow.
+
+"""
+
+# sphinx_gallery_thumbnail_path = "../../gallery/assets/optical_flow_thumbnail.png"
```

Comment:
We might need to remove the thumbnail, I'm not sure we have the rights of the `optical_flow_thumbnail.png ` image that was uploaded. But I'm sure we can use on of the images that this example will generate!

Reply:
Yes it's pending. Haven't worked much on this yet, will try to complete over this week.